desktop/config.coffee

```diff
@@ -97,6 +97,7 @@ module.exports =
   SITEMAP_BASE_URL: 'http://artsy-sitemaps.s3-website-us-east-1.amazonaws.com'
   STRIPE_PUBLISHABLE_KEY: null
   TEAM_BLOGS: '^\/life-at-artsy$|^\/artsy-education$|^\/gallery-insights$'
+  TARGET_CAMPAIGN_URL: '/seattle-art-fair-2017'
```

Comment:
Nah, this is good. The values here are just used as development defaults. So it's nice to have something in there to serve as an example when appropriate.